jetty-osgi/jetty-osgi-boot-jsp/pom.xml

```diff
@@ -56,9 +56,77 @@
+            <Import-Package>
+              org.eclipse.jdt.*;resolution:=optional,
+              org.eclipse.jdt.core.compiler.*;resolution:=optional,
+              com.sun.el;resolution:=optional,
+              com.sun.el.lang;resolution:=optional,
+              com.sun.el.parser;resolution:=optional,
+              com.sun.el.util;resolution:=optional,
+              javax.el;version="[3.0,3.1)",
+              javax.servlet;version="[3.1,4.1)",
+              javax.servlet.resources;version="[3.1,4.1)",
```

Comment:
Shouldn't the start of Servlet range be 4.0?

Reply:
For this and the other range issues you've raised we need to address this in another PR, not this one. If you're happy with the changes to enable jetty-slf4j-impl to work in osgi, please give this issue the thumbs up and open new issue to bump up osgi servlet/jsp api versions.

Reply:
Fair enough.
I opened issue #4736 for this.